Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Themes/ToggleBackgroundUgui.asset

```diff
@@ -0,0 +1,171 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!114 &11400000
+MonoBehaviour:
```

Comment:
There is discrepancies between using UnityUI vs UGUI in files in MRTK. I know UGUI is shorter but I believe Unity UI is the official name.